src/cli/tideliftMeUpCli.ts

```diff
@@ -61,12 +62,24 @@ export async function tideliftMeUpCli(args: string[]) {
 		);
 	}
 
-	const packageEstimates = await tideliftMeUp({
-		ownership,
-		since,
-		status: status as PackageStatus,
-		username,
-	});
+	try {
+		const packageEstimates = await tideliftMeUp({
+			ownership,
+			since,
+			status: status as PackageStatus,
+			username,
+		});
 
-	reporters[reporter](packageEstimates);
+		if (packageEstimates instanceof TideliftMeUpError) {
+			throw packageEstimates;
```

Comment:
The test reporter is right, this line is dead code. `tideliftMeUp` can never return an error at all so there's never a time that `packageEstimates` is actually a `TideliftMeUpError`.

This is all in agreement with https://github.com/JoshuaKGoldberg/tidelift-me-up/pull/591#discussion_r1962587786 🚀